src/System.Management.Automation/FormatAndOutput/common/PSStyle.cs

```diff
@@ -516,6 +638,17 @@ private PSStyle()
             FileInfo = new FileInfoFormatting();
         }
 
+        private static string ValidateNoContent(string text)
+        {
+            var decorartedString = new StringDecorated(text);
+            if (decorartedString.ContentLength > 0)
+            {
+                throw new InvalidOperationException(string.Format(PSStyleStrings.TextContainsContent, decorartedString.ToString(OutputRendering.PlainText)));
```

Comment:
This might arguably be an `ArgumentException` rather than an `InvalidOperationException`

Reply:
will change